app/ui/about/stars.html

```diff
@@ -29,5 +29,22 @@
 
 <body>
 Hello, 🌟.
+
+<ul id="bookmarks"></ul>
```

Comment:
I feel the front-end folks will want to set some patterns for this, but I'm cool with this approach.